src/js/components/modal.js

```diff
@@ -258,6 +258,9 @@ var Modal = (function() {
     if (document.activeElement === first) {
       event.preventDefault();
       last.focus();
+    } else if (document.activeElement === activeModal) {
+      event.preventDefault();
+      first.focus();
```

Comment:
I would say it makes more sense to move focus to the last focusable element when the user presses `Shift + Tab` first.  So `last.focus();` instead.

Reply:
This has been updated 👍